### PR TITLE
Minor fix for header

### PR DESCRIPTION
**Pull request by Yustynn**

### CODE REVIEW

**Comment by LanaCoyote** [comment]
Great! Thanks for moving the gus logo for the homepage!

File: browser/js/directives/gg-header/gg-header.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <header>
-  <img src='/assets/images/gus.png' id='logo' />
+  <img src='/images/gus.png' id='logo' />
```
Comment (LanaCoyote) [inline browser/js/directives/gg-header/gg-header.html:2]: :balloon: :tada:

**Comment by LanaCoyote** [comment]
Oh wait a second, I wasn't totally done reading through. Why did we remove user auth??

**Comment by LanaCoyote** [comment]
Wait nevermind I'm a dumbass you deleted the public folder >_>

**Comment by Yustynn** [comment]
Mmmhmm. Shouldn't have been there in the first place

**Pull request merged**